blog/2025-07-cluster-api-provider-gardener.md

```diff
@@ -0,0 +1,129 @@
+
+Gardener and Cluster API both aim to automate Kubernetes cluster management, but they approach the problem from opposite directions and with different user audiences in mind.
+
+#### Gardener API: Platform-first and top-down
+
+Gardener is a top-down architected platform, designed to deliver homogeneous Kubernetes clusters across clouds as a Service. It was built with application or service teams in mind who require consistent, production-ready Kubernetes clusters without managing internal details.
+- A single manifest defines an entire cluster (`Shoot`) with platform-chosen defaults.
+- Users benefit from a user experience layer, sensible defaults, and well-integrated extensions (e.g. DNS, worker pools, cloud-specific settings).
+- Cluster lifecycle is abstracted away: versioning, networking, and OS images are handled centrally.
+- Clusters can be managed for multiple cloud providers simultaneously.
+
+This setup is depicted in the following illustration, where the Gardener API is used to manage clusters as a service:
+
+![Illustration: Gardener managed by service team](./static/gardener-managed-by-service-team.svg)
+
+#### Cluster API: Infrastructure-centric and bottom-up
+
+Cluster API (CAPI), on the other hand, is a bottom-up framework for building cluster management solutions. It’s aimed at infrastructure teams who need fine-grained control over every component of a Kubernetes cluster and know how to assemble them.
+- A cluster is defined via multiple resource manifests (Cluster, MachineDeployment, KubeadmConfig, etc.).
+- Each resource is reconciled by dedicated controllers.
+- There’s no opinionated user interface. Users must understand the internals of Kubernetes cluster bootstrapping.
+- Powerful for building your own platform, but requires substantial operational ownership.
+
+This setup is illustrated below, where the end user manages the full lifecycle of the Cluster API management plane:
+
+![Illustration: Cluster API self-managed](./static/cluster-api-self-managed.svg)
+
+You can think of Gardener as a second evolutionary stage of Cluster API: more opinionated, more integrated, and focused on platform-level concerns.
+
+### When to use which?
+
+|                      | Cluster API                          | Gardener API                               |
+|----------------------|--------------------------------------|--------------------------------------------|
+| Target audience      | Infrastructure / platform engineers  | Application / service teams                |
+| Interface style      | Multiple manifests, component-driven | One manifest, opinionated, platform-driven |
+| Ownership            | End user manages infrastructure      | Platform team manages infrastructure       |
+| Multi-cloud support  | Configure it yourself                | Built-in                                   |
+| Lifecycle automation | Assemble from primitives             | Out-of-the-box automation                  |
+
```

Comment:
move segment "Cluster API: Managed by End User" and "Gardener API + CAPGa: Managed by Service Teams" here. They actually discuss more in depth the reasons for the above table.